Sources/StreamVideoSwiftUI/CallView/VideoParticipantsView.swift

```diff
@@ -361,7 +361,7 @@ public struct VideoCallParticipantView: View {
                     }
                 }
             )
-            .id(participant.id)
+//            .id(participant.renderingId)
```

Comment:
why do we remove it?

Reply:
That was to ensure that tracks aren't disappearing because views where stuck in a previous superview. I have reverted it.